src/libs/fullstory/index.native.ts

```diff
@@ -0,0 +1,59 @@
+import FullStory, {FSPage} from '@fullstory/react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import type Session from '@src/types/onyx/Session';
+import type {UserSession} from './types';
+
+/**
+ * Fullstory React-Native lib adapter
+ * Proxy function calls to React-Native lib
+ * */
+const FS = {
+    /**
+     * Sets the identity as anonymous using the FullStory library.
+     */
+    anonymize: () => FullStory.anonymize(),
+
+    /**
+     * Sets the identity consent status using the FullStory library.
+     */
+    consent: (c: boolean) => FullStory.consent(c),
+
+    /**
+     * Initializes the FullStory session with the provided session information.
+     */
+    consentAndIdentify: (value: OnyxEntry<Session>) => {
+        try {
+            const session: UserSession = {
+                email: value?.email,
+                accountID: value?.accountID,
+            };
+            // set consent
+            FullStory.consent(true);
+            FS.fsIdentify(session);
+        } catch (e) {
+            // error handler
```

Comment:
NAB seems like we're not properly handling errors here. I think this is fine for now, but maybe we should log something here.